main.py

```diff
@@ -56,7 +56,11 @@ def rePhone(fname):
   #extract the specific phone numbers regex
   #if the length of the extraction is not empty
   #print the phone number
-  pass
+  fhand = open(fname,'r')
+  for line in fhand:
+      extr = re.findall('\s(.*-.*-.*)',line)
```

Comment:
A phone number should have digits, not just any character.

`extr = re.findall('\d{3}-\d{3}-\d{4}', line)`